src/js/components/charts/DialChart.js

```diff
@@ -80,14 +78,20 @@ var DialChart = React.createClass({
     }
 
     // Zero-length defaults are populated with actual data if available
-    var namedSlices = _.indexBy(slices, 'name');
-    var namedData = _.indexBy(data, 'name');
-    var normalizedNamedData = _.extend(namedSlices, namedData);
-    return _.values(normalizedNamedData);
+    var namedSlices = slices.reduce(function (indexByName, slice) {
+      indexByName[slice.name] = slice;
+      return indexByName;
+    }, {});
+    var namedData = data.reduce(function (indexByName, datum) {
+      indexByName[datum.name] = datum;
+      return indexByName;
+    }, {});
+    var normalizedNamedData = Object.assign({}, namedSlices, namedData);
+    return Object.values(normalizedNamedData);
```

Comment:
Please add newlines before `return` statements. It is only if `return` is the only line in that block we don't add a newline before.